package.json

```diff
@@ -4,14 +4,12 @@
   "description": "OpenTelemetry is a distributed tracing and stats collection framework.",
   "scripts": {
     "precompile": "lerna run version",
-    "compile": "lerna run protos && tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "compile": "lerna run protos --concurrency=1 && tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
```

Comment:
`lerna` runs the scripts in parallel. This here is done to ensure that the two `git` processes (one from `otlp-proto-exporter-base` and one from `otlp-grpc-exporter-base`) do not interfere with each other.